File: JS/UberX.js
class UberX extends Car {

    constructor(license,driver,brand,model){
        super(license,driver)
        this.brand = brand;
        this.brandmodel = model;
    }

    printDatacar() {
        console.log(this.driver)
        console.log(this.driver.name)
        console.log(this.driver.document)
        }
}

File: Java/UberX.java
package Java;
class UberX extends Car{
    String brand;
    String model;

    public UberX(String license,Account driver,String brand,String model){
        super(license,driver);
        this.brand = brand;
        this.model = model;
    }
    void printDataCar(){
        if(getPassengenger()>=4){
            System.out.println("-- License: "+license +"  -- Driver: "+driver.name+"  -- numero de pasajeros : "+getPassengenger());
        }
    }
}